### ISSUE TITLE
Formatting template literals glitch

### ISSUE THREAD

**Issue by abonae** [sunjw/jstoolnpp#71]
Hi
Firstly great tool - many thanks.

The formatter changes the content of template literals in some situations, which it should never do.

For example
```   
    some code;
    const foo = `alkjd
alskdjlakjd
`;
    some more code
```
is formatted into
```
    some code;
    const foo = `alkjd
alskdjlakjd
        `;
    some code;
```
which has added spaces to indent the closing \`, but in so doing has changed the value of the string literal.  String literals do look bad from an indentation perspective, but they are what they are.  Changing the value, and hence the program's behaviour, is an error.

Appreciate this being added to the list of issues to fix.

Thanks
Abo

**Comment by sunjw** [comment]
Thanks for reporting this.
It will be fixed soon.

**Comment by abonae** [comment]
Thanks!

**Comment by sunjw** [comment]
Fixed in a1708d66320719be0344b7fbbf45a9b00c5a2b1a
